### ISSUE TITLE
Benchmarking fails for VerusHash and RandomX with nanominer

### ISSUE THREAD

**Issue by Boulder08** [RainbowMiner/RainbowMiner#2735]
Both do nothing in RBMiner's benchmarking.

Running nanominer.exe config_NiceHash-RandomX-CPU.txt manually works fine.
nanominer.exe config_NiceHash-VerusHash-CPU.txt fails, it just does nothing and stops without any specific error message.

**Comment by whitindllc** [comment]
do any of your other nicehash benchmarks work?

**Comment by Boulder08** [comment]
Yes, I've not noticed any issues with Nicehash other than this one. Xmrig has been doing RandomX quite often in Nicehash so it's benchmarked fine.

**Comment by whitindllc** [comment]
it looks like it supports it on CPU but my pc doesnt even try to mine with it... perhaps the dev fee is too high
![image](https://github.com/RainbowMiner/RainbowMiner/assets/128111946/65e82986-b2ff-49de-b75a-7956150718a2)

looks like my pc fails on nicehash but works fine with nanominer over at herominers... it also looks like xmrig is about 25% faster
